Governance_v_1_6 (#304)
MinBZK/Algoritmekader: docs/maatregelen/0-org-04-politiek-bestuurlijke-verantwoordelijkeheid.md

Reviewer: De maatregel en de toelichting lopen hier nog door elkaar heen. De maatregel is normaal gesproken in 1 of 2 zinnen en beschrijft echt een concrete actie. In de toelichting staat dan waarom dat nodig is en hoe je dat kan doen.

Reviewer: De maatregel is misschien:

Zorg voor politiek en bestuurlijke bewustwording van de verantwoordelijkheden rondom de inzet van algoritmes. Zorg dat bestuurders bewust zijn van de voor- en nadelen van de inzet van algoritmes en daarnaar kunnen handelen.

Author: Aangepast

---

> title: Visie en politiek-bestuurlijke verantwoordelijkheid
> toelichting: Zorg voor een organisatievisie en politiek-bestuurlijk bewustzijn.

Reviewer: ```suggestion
title: Visie en politiek-bestuurlijke verantwoordelijkheid
id: urn:nl:ak:mtr:org:04
toelichting: Zorg voor een organisatievisie en politiek-bestuurlijk bewustzijn.
```

levenscyclus:
- organisatieverantwoordelijkheden
rollen:
- projectleider
# - beleid-en-advies
---
<!-- tags -->

> ## Maatregel
> Voor een passende algoritmegovernance is politiek-bestuurlijk bewustzijn, betrokkenheid en verantwoordelijkheid essentieel. 
> De kernvraag voor publieke organisaties bij de inzet van algoritmen is altijd: Hoe wegen we (als publieke organisatie of samenleving) de voordelen en nadelen van de inzet van algoritmen? 
> Dit is per definitie een kwalitatieve en politieke vraag. 
> Dit gaat niet alleen over direct opbrengsten maar ook over lange termijn en indirecte effecten, de mate waarin de inzet van technologie bijdraagt aan de legitimiteit van publieke organisatie en hoe burgers met deze technologie worden bejegend. 
>  
> Een politiek-bestuurlijk kader is vervolgens om te zetten tot een uitgeschreven visie en aanpak om dit te concretiseren. Zie voorbeelden.

Reviewer: ```suggestion
## Maatregel
Zorg voor politiek en bestuurlijke bewustwording van de verantwoordelijkheden rondom de inzet van algoritmes. Zorg dat bestuurders bewust zijn van de voor- en nadelen van de inzet van algoritmes en daarnaar kunnen handelen.

## Toelichting
Voor een passende algoritmegovernance is politiek-bestuurlijk bewustzijn, betrokkenheid en verantwoordelijkheid essentieel. 
De kernvraag voor publieke organisaties bij de inzet van algoritmen is altijd: Hoe wegen we (als publieke organisatie of samenleving) de voordelen en nadelen van de inzet van algoritmen? 
Dit is per definitie een kwalitatieve en politieke vraag. 
Dit gaat niet alleen over direct opbrengsten maar ook over lange termijn en indirecte effecten, de mate waarin de inzet van technologie bijdraagt aan de legitimiteit van publieke organisatie en hoe burgers met deze technologie worden bejegend. 
 
Een politiek-bestuurlijk kader is vervolgens om te zetten tot een uitgeschreven visie en aanpak om dit te concretiseren.
```

> ## Aandachts- en actiepunten

Reviewer: Dit hoofdstuk hebben we bij geen enkele maatregel. Daarom weghalen. De actie moet in de maatregel zelf. Je kan het eventueel verder uitsplitsen in de toelichting. 

Misschien is dus het opstellen van een visie of strategie ook een maatregel?

Author: Wat bedoel je met "Dit hoofdstuk hebben we bij geen enkele maatregel"? 

Eens dat het uit te splitsen is in politiek-bestuurlijk en visie, maar nu geen tijd voor.

* Op politiek-bestuurlijk niveau bewustzijn en een weloverwogen visie is belangrijk om verantwoorde AI en algortimes in te kunnen zetten. Veel pilots en initiatieven lopen hierop stuk door hierop vooruit te lopen.
* Een meerjarenbeleidsplan of een strategie- of visedocument kan richting geven in helpen met latere afwegingen maken.

>     * Eerst een AI-casus doorlopen als proef en een IAMA uitwerken kan waardevolle input hierbij zijn.

Reviewer: ```suggestion
* Eerst een AI-casus doorlopen als proef en een IAMA uitwerken kan waardevolle input hierbij zijn.
```

---
title: Politiek-bestuurlijke verantwoordelijkheid
toelichting: Zorg voor politiek-bestuurlijk bewustzijn, betrokkenheid, en verantwoordelijkheid. Zorg dat bestuurders bewust zijn van de voor- en nadelen van de inzet van algoritmes en daarnaar kunnen handelen.
levenscyclus:
- organisatieverantwoordelijkheden
rollen:
- projectleider
- beleid-en-advies
---
<!-- tags -->

## Maatregel

Zorg voor politiek-bestuurlijk bewustzijn, betrokkenheid, en verantwoordelijkheid. Zorg dat bestuurders bewust zijn van de voor- en nadelen van de inzet van algoritmes en daarnaar kunnen handelen.

## Toelichting

Voor een passende algoritmegovernance is politiek-bestuurlijk bewustzijn, betrokkenheid en verantwoordelijkheid essentieel. 
De kernvraag voor publieke organisaties bij de inzet van algoritmen is altijd: Hoe wegen we (als publieke organisatie of samenleving) de voordelen en nadelen van de inzet van algoritmen? 

> Dit is per definitie een kwalitatieve en politieke vraag. 

Reviewer 2: ```suggestion
Dit is per definitie een kwalitatieve en politieke vraag.
```

Dit gaat niet alleen over direct opbrengsten maar ook over lange termijn en indirecte effecten, de mate waarin de inzet van technologie bijdraagt aan de legitimiteit van publieke organisatie en hoe burgers met deze technologie worden bejegend. 
 
Om te zorgen voor politiek-bestuurlijke betrokkenheid kan het helpen om een meerjarige visie en/of strategie rondom verantwoorde inzet te formulueren. 

> * Op politiek-bestuurlijk niveau bewustzijn en een weloverwogen visie is belangrijk om verantwoorde algoritmes in te kunnen zetten. Veel pilots en initiatieven lopen hierop stuk door hierop vooruit te lopen.

Reviewer 2: ```suggestion
* Een weloverwogen visie op politiek-bestuurlijk niveau is belangrijk om verantwoorde algoritmes in te kunnen zetten. Dit ondersteunt teams bij het maken van de juiste overwegingen bij de ontwikkeling en gebruik van algoritmes. 
```
